app/jobs/add_works_to_collections_job.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class AddWorksToCollectionsJob < ApplicationJob
+  queue_as :relationships
+
+  def perform(record_data:)
+    work = record_data[:model].constantize.where(identifier: record_data[:identifier]).first
+    collection = Collection.where(identifier: record_data[:parents]).first
+
+    collection.try(:reindex_extent=, Hyrax::Adapters::NestingIndexAdapter::LIMITED_REINDEX)
+    work.member_of_collections << collection
+    work.save!
+    Rails.logger.info("🦄🪺👩‍👩‍👧‍👦 #{record_data[:identifier]} processed")
+  rescue StandardError => e
+    notice = "😈😈😈 ERROR: unable to create relationship for #{record_data[:identifier]}"
+    Rails.logger.error(notice)
+    Rails.logger.error(e.message)
+    File.open('lib/data/add_works_to_collection_errors.txt', 'a') do |f|
```

Comment:
This needs to be saved to a persisted volume director: 

![image](https://github.com/scientist-softserv/adventist-dl/assets/10081604/2d4600f6-3d16-4b30-8009-a1bb66fb2215)